esx_service/utils/error_code.py

```diff
@@ -46,11 +46,18 @@ class ErrorCode:
     # DATASTORE related error code start
     DEFAULT_DS_NOT_SET = 301
     DS_NOT_EXIST = 302
+
+    # indicates that config init needed for a requested operation
```

Comment:
Nit: it seems some of our comments start with capital letter, some don't. Let's keep them consistent.